src/scalar_mul.rs

```diff
@@ -130,6 +148,17 @@ impl NonAdjacentForm for jubjub::Scalar {
     }
 }
 
+impl NonAdjacentForm for jubjub::Scalar {
+    fn inner_to_bytes(&self) -> [u8; 32] {
+        self.to_bytes()
+    }
+
+    /// The NAF length for Jubjub is 253 since Jubjub's order is about 2<sup>251.85</sup>.
```

Comment:
I used the tag since this is how it renders in the docs 
![image](https://user-images.githubusercontent.com/8863307/234109611-0ac8870f-df20-434f-a4e6-92b6f9915b77.png)

Whereas the caret renders like this ![image](https://user-images.githubusercontent.com/8863307/234109038-1d020ace-1e5b-4a57-b763-36bcf9587cc2.png)

Should I still commit the suggestion?

There's a third way of doing it, and that is pulling something like Katex into the HTML header of the docs via a compilation option, and then using regular latex.

Reply:
Yeah that looks nice. OK ignore me :D